resources/MrMathTables.js

```diff
@@ -0,0 +1,2289 @@
+"Gloves":{"zone":"hand","coverage":["Ha"]},
+"Gauntlets":{"zone":"hand","coverage":["Ha"]},
+"Mittens":{"zone":"hand","coverage":["Ha"]},
+"Shorts":{"zone":"thor","coverage":["Hp", "Gr","Th"]},
+"Loincloth":{"zone":"thor","coverage":["Hp","Gr"]},
+"Skirt(mini)":{"zone":"thor","coverage":["Hp","Gr"]},
+"Skirt(short)":{"zone":"thor","coverage":["Hp","Gr","Th"]},
+"Skirt(mid)":{"zone":"thor","coverage":["Hp","Gr","Th","Kn"]},
+"Skirt(long)":{"zone":"thor","coverage":["Hp","Gr","Th","Kn","Ca"]},
+"Pants":{"zone":"legs","coverage":["Hp","Gr","Th","Kn","Ca"]},
+"Leggings":{"zone":"legs","coverage":["Hp","Gr","Th","Kn","Ca","Ft"]},
+"Boots, calf":{"zone":"feet","coverage":["Ca","Ft"]},
+"Boots, knee":{"zone":"feet","coverage":["Kn","Ca","Ft"]},
+"Shoes":{"zone":"feet","coverage":["Ft"]},
+"Sandals":{"zone":"feet","coverage":["Ft"]},
+"Gorget":{"zone":"head","coverage":["Nk"]},
+"Ailettes":{"zone":"aile","coverage":["Sh"]},
+"Rerebraces":{"zone":"rere","coverage":["Ua"]},
+"Coudes":{"zone":"coud","coverage":["El"]},
+"Vambraces":{"zone":"vamb","coverage":["Fo"]},
+"Kneecops":{"zone":"knee","coverage":["Kn"]},
+"Greaves":{"zone":"grev","coverage":["Ca"]}}
+
+var armor_prot = {"nat1":["nat1","1","1","1","1"],
+"nat2":["nat2","2","2","2","2"],
+"nat3":["nat3","3","3","3","3"],
+"fur(light)":["fur","3","2","2","3"],
+"fur(thick)":["fur","5","3","2","4"],
+"cloth":["cloth","1","1","1","1"],
+"canvas":["cloth","1","1","1","1"],
+"waxed canvas":["cloth","1","1","1","1"],
+"silk":["cloth","1","1","1","1"],
+"linen":["cloth","1","1","1","1"],
+"lightsilk":["cloth","0","1","0","0"],
+"lightlinen":["cloth","0","1","0","1"],
+"serge":["cloth","1","1","1","1"],
+"buckram":["cloth","1","1","1","1"],
+"russet":["cloth","1","1","1","1"],
+"worsted":["cloth","1","1","1","1"],
+"quilt":["quilt","5","3","2","4"],
+"quilt+1":["quilt+1","6","4","3","5"],
+"sealskin":["quilt","5","3","2","5"],
+"leather":["leather","2","4","3","3"],
+"leather+1":["leather+1","3","5","4","4"],
+"leather+2":["leather+2","4","6","5","5"],
+"beaver":["leather","2","4","3","3"],
+"ermine":["leather","2","4","3","3"],
+"kurbul":["kurbul","4","5","4","3"],
+"kurbul+1":["kurbul+1","5","6","5","4"],
+"ring":["ring","3","6","4","3"],
+"ring+1":["ring+1","4","7","5","4"],
+"ring+2":["ring+2","5","8","6","5"],
+"mail":["mail","2","8","5","1"],
+"mail+1":["mail+1","3","9","6","2"],
+"mail+2":["mail+2","4","10","7","3"],
+"mail+3":["mail+3","5","11","8","4"],
+"scale":["scale","5","9","4","5"],
+"plate":["plate","6","10","6","2"],
+"plate+1":["plate+1","7","11","7","3"],
+"plate+2":["plate+2","8","12","8","4"],
+"bone":["bone","2","2","2","2"],
+"bone+1":["bone+1","3","3","3","3"],
+"bone+2":["bone+2","4","4","4","4"],
+"gargun":["nat","4","3","1","3"],
+"horse":["nat","4","3","1","3"],
+"dog":["nat","4","3","1","3"],
+"wolf":["nat","4","3","1","3"],
+"Dire wolf":["nat","5","4","2","4"],
+"Brown bear":["nat","6","4","3","5"],
+"belt lizard king":["magic belt","2","4","3","3"]
```

Comment:
"Lizard king belt"?  I think that this exposes the need for house rule additional armor tables. 😄

Reply:
Created issue #36

Reply:
created #36